ncs/Kconfig

```diff
@@ -29,3 +29,27 @@ config SUIT_ENVELOPE_OUTPUT_MPI_MERGE
 config SUIT_LOCAL_ENVELOPE_GENERATE
   bool "Generate local envelope"
   default y if SOC_NRF54H20_CPUAPP || SOC_NRF54H20_CPURAD
+
+config SUIT_EXTRACT_IMAGE_TO_DFU_CACHE
+  bool "Extract firmware image to DFU cache"
+  help
+    Extracts the firmware image to a DFU cache file, which can be then flashed separately
+    to the device (instead of being integrated into the SUIT envelope). If using the default
+    SUIT envelope template, this will also remove the firmware image from the SUIT envelope
+    integrated payloads.
+
+if SUIT_EXTRACT_IMAGE_TO_DFU_CACHE
+
+config SUIT_IMAGE_DFU_CACHE_PARTITION
+  int "The number of the DFU partition to which the image will be extracted"
+  help
+    This option will ensure that images which set it to the same number will be extracted
+    to the same dfu cache file.
+  default 1
+
+config SUIT_IMAGE_DFU_CACHE_URI
+  string "The URI used as key for the image in the DFU cache"
+  default "cache://application" if SOC_NRF54H20_CPUAPP
+  default "cache://radio" if SOC_NRF54H20_CPURAD
```

Comment:
To my understanding - Sylwester's idea was to encode the board/SoC/other identification to avoid name conflicts in case of external (i.e. cloud) sources.

The most fancy thing I can image is:
```
default  "cache://samples/$(shell,echo ${APPLICATION_SOURCE_DIR##*/})/${APP_VERSION_EXTENDED_STRING}/${BOARD}/application.bin"
```

Reply:
Turned out there are a bit too many problems with it, so currently leaving it in a simpler vrsion.

The problems: `$(shell,echo ${APPLICATION_SOURCE_DIR##*/})` - probably problems with portability, addtitionally this would not work for radio core where this would resolve to `hci_ipc`

`${APP_VERSION_EXTENDED_STRING}` - would just work for application core, the version is not accessible from radio core